tests/io_components/test_metapartition.py

```diff
@@ -1452,17 +1453,18 @@ def test_concat_metapartition_different_partitioning(df_all_types):
         MetaPartition.concat_metapartitions([mp1, mp2])
 
 
-def test_partition_on_scalar_intermediate(df_not_nested):
+# We can't partition on null columns (gh-262)
+@pytest.mark.parametrize("col", set(get_dataframe_not_nested().columns) - {"null"})
```

Comment:
This results in a not deterministic test collection which is usually not a good thing. We should make this deterministc by using `sorted(set(...`

It also breaks execution with pytest-xdist